test-framework/sudo-compliance-tests/src/sudo/flag_shell.rs

```diff
@@ -67,6 +67,22 @@ echo $0";
     Ok(())
 }
 
+#[ignore = "gh962"]
+#[test]
+fn shell_is_canonicalized() -> Result<()> {
+    let shell_path = "/bin/bash";
+    let env = Env("ALL ALL=(ALL:ALL) NOPASSWD: /usr/bin/bash")
```

Comment:
Maybe create a symlink ourselves here? This won't work on FreeBSD as `/bin` is not a symlink to `/usr/bin`.